api/ClipboardItem.json

```
@@ -67,10 +67,14 @@
           "description": "<code>ClipboardItem()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "66"
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data. Full implementation would also allow for a string or a Promise which resolves with either a string or blob. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
```

If this is ever fixed, the note won't age very well. Suggesting a rewrite to make it easier to prepend "Before version X" to the note, when the time comes. What do you think of this version?

```suggestion
              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data, but not strings or Promises that resolve to strings or blobs. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
```

---

Absolutely - I hadn't considered that - learning all the time ☺️